website/client/src/assets/scss/button.scss

```diff
@@ -28,47 +28,133 @@
 
   &:disabled, &.disabled {
     cursor: default;
-    opacity: 0.75 !important;
+    opacity: 0.75;
   }
 }
 
+.btn-front {
+  font-size: 16px;
+  line-height: 1.5;
+  padding: 7.5px 15.5px;
+}
+
 .btn-primary {
-  background: $purple-200 !important;
+  background: $purple-200;
+  border: 1px solid transparent;
+
+  &:hover {
+    background: #5d3b9c;
+    border: 1px solid transparent;
+  }
+
+  &:focus {
+    background: $purple-200;
+    border-color: $purple-400;
+  }
+
+  &:not(:disabled):not(.disabled):active:focus, &:not(:disabled):not(.disabled).active:focus {
```

Comment:
I think we could use 

```scss
&:not(:disabled):not(.disabled) {
  &:active:focus, &.active:focus {

  } 
}
```

then we'd have atleast cleaner scss (it should look the same in the generated code)